exchange/exchange.go

```diff
@@ -387,22 +390,23 @@ func applyCategoryMapping(ctx context.Context, requestExt openrtb_ext.ExtRequest
 				duration = bid.bidVideo.Duration
 				category = bid.bidVideo.PrimaryCategory
 			}
-
-			if category == "" {
-				bidIabCat := bid.bid.Cat
-				if len(bidIabCat) != 1 {
-					//TODO: add metrics
-					//on receiving bids from adapters if no unique IAB category is returned  or if no ad server category is returned discard the bid
-					bidsToRemove = append(bidsToRemove, bidInd)
-					continue
-				} else {
-					//if unique IAB category is present then translate it to the adserver category based on mapping file
-					category, err = categoriesFetcher.FetchCategories(ctx, primaryAdServer, publisher, bidIabCat[0])
-					if err != nil || category == "" {
+			if includeBrandCategory {
+				if category == "" {
```

Comment:
nitpicky: This could be combined to `if includeBrandCategory && category == "" {` to reduce nesting level.